packages/salesforcedx-vscode-apex/package.json

```diff
@@ -70,7 +70,8 @@
       "test": [
         {
           "id": "sfdx.force.test.view",
-          "name": "%test_view_name%"
+          "name": "%test_view_name%",
+          "when": "sfdx:project_opened"
```

Comment:
This was a recently added bug fix from VS Code (see https://github.com/Microsoft/vscode/issues/49145)

The `sfdx:project_opened` comes from the main core extension.